docs/resources/integration.md

```diff
@@ -0,0 +1,72 @@
+
+* [GET /api/v2/integrations](https://developer.mypurecloud.com/api/rest/v2/integrations/#get-api-v2-integrations)
+* [POST /api/v2/integrations](https://developer.mypurecloud.com/api/rest/v2/integrations/#post-api-v2-integrations)
+* [GET /api/v2/integrations/{integrationId}](https://developer.mypurecloud.com/api/rest/v2/integrations/#get-api-v2-integrations--integrationId-)
+* [DELETE /api/v2/integrations/{integrationId}](https://developer.mypurecloud.com/api/rest/v2/integrations/#delete-api-v2-integrations--integrationId-)
+* [PATCH /api/v2/integrations/{integrationId}](https://developer.mypurecloud.com/api/rest/v2/integrations/#patch-api-v2-integrations--integrationId-)
+* [GET /api/v2/integrations/{integrationId}/config/current](https://developer.mypurecloud.com/api/rest/v2/integrations/#get-api-v2-integrations--integrationId--config-current)
+* [PUT /api/v2/integrations/{integrationId}/config/current](https://developer.mypurecloud.com/api/rest/v2/integrations/#put-api-v2-integrations--integrationId--config-current)
+* [GET /api/v2/integrations/types/{typeId}](https://developer.mypurecloud.com/api/rest/v2/integrations/#get-api-v2-integrations-types--typeId-)
+* [GET /api/v2/integrations/types/{typeId}/configschemas/{configType}](https://developer.mypurecloud.com/api/rest/v2/integrations/#get-api-v2-integrations-types--typeId--configschemas--configType-)
+
+
+## Example Usage
+
+```terraform
+resource "genesyscloud_integration" "integration1" {
+  intended_state   = "DISABLED"
+  integration_type = "embedded-client-app"
+  config {
+    name = "test_integration name"
+    properties = jsonencode({
+      "displayType" : "standalone",
+      "sandbox" : "allow-scripts,allow-same-origin,allow-forms,allow-modals",
+      "url" : "https://mypurecloud.github.io/purecloud-premium-app/wizard/index.html"
+      # Potential groups and queues filter (Need to look up the key name from integration type schema)
+      "groups" : [genesyscloud_group.test_group.id]
+    })
+    advanced = jsonencode({})
+    notes    = "Test config notes"
+    credentials {
+      basic_Auth = genesyscloud_credentials.test_cred.id
+    }
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- **integration_type** (String) Integration type.
+
+### Optional
+
+- **config** (List of Object) Integration config. Each integration type has different schema, use [GET /api/v2/integrations/types/{typeId}/configschemas/{configType}](https://developer.mypurecloud.com/api/rest/v2/integrations/#get-api-v2-integrations-types--typeId--configschemas--configType-) to check schema, then use the correct attribute names for properties. (see [below for nested schema](#nestedatt--config))
+- **id** (String) The ID of this resource.
+- **intended_state** (String) Integration state (ENABLED | DISABLED | DELETED). Defaults to `DISABLED`.
+
+<a id="nestedatt--config"></a>
+### Nested Schema for `config`
+
+Optional:
+
+- **advanced** (String)
```

Comment:
The descriptions here are missing because `schema.SchemaConfigModeAttr` is set on the `config` attribute. I think you can just remove `schema.SchemaConfigModeAttr` as it isn't needed here.

Reply:
Removed. Now it has descriptions for the config.